test/modules/EdgeHubRestartTester/src/EdgeHubConnectorConfig.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace EdgeHubRestartTester
+{
+    using System;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    class EdgeHubConnectorConfig
+    {
+        public EdgeHubConnectorConfig(
```

Comment:
I suggest to rename since this class contains config for both direct method and message connector.
```suggestion
        public EdgeHubConnectorsConfig(
```